test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightDeltaReaderTests.cs

```diff
@@ -80,17 +80,21 @@ public class ODataJsonLightDeltaReaderTests
 
         #region ODataV4 tests
 
-        [Fact]
-        public void ReadExample30FromV4Spec()
+        [InlineData(/*isResponse*/true)]
+        [InlineData(/*isResponse*/false)]
+        [Theory]

```

Comment:
>[Theory] [](start = 8, length = 8)

Maybe it's better to put [Theory] on top?

Reply:
Okay; sure.

---
In reply to: [256642911](https://github.com/OData/odata.net/pull/1404#discussion_r256642911) [](ancestors = 256642911)